source/Ship.cpp

```diff
@@ -3355,10 +3365,17 @@ double Ship::HeatDissipation() const
 // Get the maximum heat level, in heat units (not temperature).
 double Ship::MaximumHeat() const
 {
-	return MAXIMUM_TEMPERATURE * (cargo.Used() + attributes.Mass() + attributes.Get("heat capacity"));
+	return attributes.Get("maximum temperature") / 10. * (cargo.Used() + attributes.Mass()
+			+ attributes.Get("heat capacity"));
 }
 
 
+// Get the ship temperature.
+double Ship::ShipTemperature() const
+{
+	return attributes.Get("maximum temperature") * Heat();
```

Comment:
so, why is the ship's temperature, it's maximum temperature times it's actual temperature?

Reply:
oh wait, heat is in percent right?